models/product.model.lkml

```diff
@@ -3,6 +3,23 @@ connection: "snowflake"
 # Limit include only to the ones really needed by this explore
 include: "/views/marts/product/*.view.lkml"
 
+explore: fct_nps_feedback {
+  label: "NPS Feedback"
+  group_label: "[New] NPS Feedback"
+
+  join: dim_cloud_customers {
+    relationship: one_to_one
+    type: left_outer # We might not have customer telemetry for the NPS feedback server
+    sql_on: ${fct_nps_feedback.server_id} = ${dim_cloud_customers.server_id} ;;
+  }
+
+  join: dim_self_hosted_customers {
+    relationship: one_to_one
```

Comment:
Similar to the previous. Shouldn't this be `many_to_one`?

Reply:
Updated to `many_to_one`